src/engine/GraphStoreProtocol.h

```diff
@@ -85,6 +85,8 @@ class GraphStoreProtocol {
     updateClause::GraphUpdate up{std::move(convertedTriples), {}};
     ParsedQuery res;
     res._clause = parsedQuery::UpdateClause{std::move(up)};
+    res._originalString =
+        "Unavailable for SPARQL Graph Store Protocol POST operation";
```

Comment:
Having the same operation string for different operations is problematic, when we use this operation string for the query registry. But for the graph store we don't always have an operation string available. In the case of large POSTs it might also be unfeasible to generate an equivalent operation string. Maybe use a UUID in that case. @joka921

Reply:
My uncertainty and the questions here boil down to the question: is it problem if the operations don't have meaningful representation? It seems that the id generation is independent from the operation string, but can this cause any other problems or is the query registry only used for the cancellation handles?